public/locales/en-US.json

```diff
@@ -29,9 +29,9 @@
     "cardHeading_Satellite Capsule": "CPU socket usage",
     "cardHeading_Satellite Server": "CPU socket usage",
     "cardHeadingDescription": "...",
-    "cardHeadingDescription_Instance-hours": "Instance hours usage in gibibyte hours",
-    "cardHeadingDescription_Storage-gibibytes": "Data storage usage in gibibyte hours",
-    "cardHeadingDescription_Transfer-gibibytes": "Gibibyte hour usage per instance",
+    "cardHeadingDescription_Instance-hours": "Instance hours usage in binary gigabyte hours",
```

Comment:
@jlprevatt confirm this changes towards
```
"cardHeadingDescription_Instance-hours": "Instance hours usage"
```

Reply:
To match the pattern, how about:
"Instance hours usage in hours"